packages/bpk-component-close-button/src/BpkCloseButton.module.scss

```diff
@@ -32,22 +32,39 @@ $button-padding: tokens.bpk-spacing-sm() * 0.5;
   padding: 0;
   border: 0;
   background-color: transparent;
-  color: tokens.$bpk-text-secondary-day;
   cursor: pointer;
   appearance: none;
 
   @include utils.bpk-touch-tappable;
 
-  @include utils.bpk-hover {
+  &__default {
     color: tokens.$bpk-text-primary-day;
-  }
+    composes: bpk-close-button;
 
-  &:active {
-    color: tokens.$bpk-text-primary-day;
+    @include utils.bpk-hover {
+      color: tokens.$bpk-text-primary-day;
+    }
+
+    &:active {
+      color: tokens.$bpk-text-primary-day;
+    }
   }
 
-  &__icon {
-    vertical-align: -1.75 * $button-padding;
-    fill: currentcolor;
+  &__onDark {
+    color: tokens.$bpk-text-primary-inverse-day;
+    composes: bpk-close-button;
+
+    @include utils.bpk-hover {
+      color: tokens.$bpk-text-primary-inverse-day;
+    }
+
+    &:active {
+      color: tokens.$bpk-text-primary-inverse-day;
+    }
   }
 }
+
+.bpk-close-button-icon {
+  vertical-align: -1.75 * $button-padding;
```

Comment:
Could we add a little comment to mention what this is doing and why we are using a negative number?

Reply:
Good question it was taken from what was already their but I can investigate.

Reply:
Was [originally](https://github.com/Skyscanner/backpack/commit/33e3e2bb6536ff82e2fa2d0ac136c633ef79810f#diff-798d68bb7917eadc4cf7be8026911c761bf1f6dfbfb29ad8ab8c4c0401ec3716) added to fix alignment in chips? I can check if that's still the case.

Reply:
The Nav bar  "[text CTA](https://percy.io/096a694a/backpack/builds/33788850/changed/1850801073?browser=chrome&browser_ids=56&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=375&widths=375%2C1280)" isn't aligned any more so I can fix that as well .